src/frontend/Page/Docs.elm

```diff
@@ -561,24 +655,194 @@ viewValueItem { author, project, version } moduleName ownerName valueName =
   in
-  li [ class "pkg-nav-value" ] [ navLink valueName url False ]
+  li [ class "pkg-nav-value" ] [ navModuleLink valueName url False ]
+
+
+
+-- VIEW ABOUT
+
+
+viewAbout : Status Project.PackageInfo -> Status Time.Posix -> Html msg
+viewAbout manifestStatus time =
+  case manifestStatus of
+    Success manifest ->
+      div [ class "block-list pkg-about" ]
+        [ h1 [ class "block-list-title" ] [ text "About" ]
+        , text manifest.summary
+        , viewInstall manifest
+        , viewRelease manifest time
+        , viewLicense manifest
+        , viewDependencies manifest
+        ]
+
+    Loading ->
+      div [ class "block-list pkg-about" ] [ text "" ] -- TODO
+
+    Failure ->
+      div
+        (class "block-list pkg-about" :: Problem.styles)
+        (Problem.offline "elm.json")
+
+
+viewInstall : Project.PackageInfo -> Html msg
+viewInstall manifest =
+  case V.fromString "0.19.0" of
+    Just elm019 ->
+      if Constraint.check elm019 manifest.elm then
+        viewInstallHelp "elm install" (Package.toString manifest.name)
+      else
+        viewInstallHelp "elm-package install" (Package.toString manifest.name)
+
+    Nothing ->
+      text "" -- TODO
+
+
+viewInstallHelp : String -> String -> Html msg
+viewInstallHelp command package =
+  div []
+    [ h1 [] [ text "Install" ]
+    , pre []
+        [ code [] [ text (command ++ " " ++ package) ]
+        ]
+    ]
+
+
+
+-- VIEW RELEASE
+
+
+viewRelease : Project.PackageInfo -> Status Time.Posix -> Html msg
+viewRelease manifest timeStatus =
+  case timeStatus of
+    Failure ->
+      text "" -- TODO
+
+    Loading ->
+      text "" -- TODO
+
+    Success time ->
+      div []
+        [ h1 [] [ text "Release" ]
+        , viewTime manifest time
+        ]
+
+
+viewTime : Project.PackageInfo -> Time.Posix -> Html msg
+viewTime manifest time =
+  let
+    releasesUrl =
+      Url.crossOrigin "https://github.com" [ Package.toString manifest.name, "releases" ] []
+  in
+  a [ href releasesUrl ]
+    [ text <|
+        String.concat
+          [ monthToString (Time.toMonth Time.utc time)
+          , " "
+          , String.fromInt (Time.toDay Time.utc time)
+          , ", "
+          , String.fromInt (Time.toYear Time.utc time)
+          ]
+    ]
+
+
+monthToString : Time.Month -> String
+monthToString month =
+  case month of
+    Time.Jan -> "January"
+    Time.Feb -> "February"
+    Time.Mar -> "March"
+    Time.Apr -> "April"
+    Time.May -> "May"
+    Time.Jun -> "June"
+    Time.Jul -> "July"
+    Time.Aug -> "August"
+    Time.Sep -> "September"
+    Time.Oct -> "October"
+    Time.Nov -> "November"
+    Time.Dec -> "December"
+
+
+
+-- VIEW LICENSE
+
+
+viewLicense : Project.PackageInfo -> Html msg
+viewLicense manifest =
+  let
+    licenseUrl =
+      Url.crossOrigin
+        "https://github.com"
+        [ Package.toString manifest.name, "blob", V.toString manifest.version, "LICENSE" ]
+        []
```

Comment:
I guess we could put the GitHub URLs in `Href.elm`. 
I didn't because the one for `Browse Source` was not and these URLs are used only once, but this could make sense.